e2e/tests/tickets.e2e.ts

```diff
@@ -0,0 +1,140 @@
+        await expect(element(by.text('New single ticket'))).toBeVisible();
+        await expect(element(by.text('New period ticket'))).toBeVisible();
+
+        // Choose single ticket
+        await element(by.text('New single ticket')).tap()
+
+        await expect(element(by.text('Single ticket'))).toBeVisible();
+        await expect(element(by.text('1 Adult'))).toBeVisible();
+        await expect(element(by.text('Travel in 1 zone (A)'))).toBeVisible();
+        await expect(element(by.text(`Total: ${ticketPrice}`))).toBeVisible();
+        await expect(element(by.text('Go to payment'))).toBeVisible();
+
+        await element(by.text('Go to payment')).tap();
+
+        await expect(element(by.text('Single ticket'))).toBeVisible();
+        await expect(element(by.label(`1 Adult ${ticketPrice} Single ticket bus/tram Valid in zone A Starting now`)).atIndex(0)).toExist()
+        await expect(element(by.text('1 Adult'))).toBeVisible();
+        await expect(element(by.text(ticketPrice)).atIndex(0)).toBeVisible();
+        await expect(element(by.text('Single ticket bus/tram'))).toBeVisible();
+        await expect(element(by.text('Valid in zone A'))).toBeVisible();
+        await expect(element(by.text('Starting now'))).toBeVisible();
+
+        /*
+        TODO How to check that 'Confirm option' is disabled before choosing a payment method?
+        await expect(element(by.text('Confirm option'))).not.toBeFocused()
+        await element(by.text('Confirm option')).getAttributes()
+            .then(e => {
+                if (!('elements' in e)) {
+                    console.log('Confirm option: ', e.text)
+                    console.log('Confirm option: ', e.value)
+                    console.log('Confirm option: ', e.enabled)
+                    console.log('Confirm option: ', e.label)
+                    console.log('Confirm option: ', e.placeholder)
+                }
+            })
+         */
+
+        // Check the payment cards
+        for (let paymentMethod of ['Visa', 'MasterCard']){
+            await element(by.text('Choose payment option')).tap();
+
+            await expect(element(by.text('Select payment option'))).toBeVisible();
+            await expect(element(by.text('Vipps'))).toBeVisible();
+            await expect(element(by.text('Visa'))).toBeVisible();
+            await expect(element(by.text('MasterCard'))).toBeVisible();
+
+            await element(by.text(paymentMethod)).tap();
+
+            await element(by.text('Confirm option')).tap();
+
+            // Validate loading of external payment component
+            await expect(element(by.text('Loading payment terminal…'))).toBeVisible();
+            await waitFor(element(by.type('AXRemoteElement'))).toExist();
+
+            // Cancel
+            await element(by.text('Cancel')).tap();
+
+            await expect(element(by.text('Single ticket'))).toBeVisible();
+
+        }
+
+        //TODO How to test the Vipps integration?
```

Comment:
Ingen godt måte egentlig, Vipps-appen kan ikke installeres på simulator.

Reply:
Da får vi la denne ligge til manuell QA 😄

Reply:
Hviiis vi kunne plukket opp at appen prøver seg på en deeplink så er det en måte. Uten at den faktisk får gjennomført det. Men er ingen skikkelig godt måte utenom manuell QA. 

En fremtidig (kanskje litt crazy) tanke er; hvis AtB kjøper seg en Mac Mini i fremtiden, og bruker denne som "bygg-agent" for Github Actions, så er det mulig å ta noen av test device labben og koble dette til Mac-en. Og så er da da mulig å kjøre E2E-tester på en faktisk device. I det tilfellet har det vært mulig å teste overgang til Vipps.

Reply:
Wooh. Ja, den var snedig. Tar med ideen videre :-)